packages/structure/src/model/RWRoute.ts

```diff
@@ -144,6 +144,10 @@ export class RWRoute extends BaseNode {
     return this.getBoolAttr('prerender')
   }
 
+  @lazy() get renderMode(): string | undefined {
```

Comment:
We decided to not do render modes right now, correct?
If so, we should remove this